packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -69,14 +75,43 @@ export default {
         },
         imgSrc: {
             type: String,
-            default: null
+            default: ''
+        },
+        altText: {
+            type: String,
+            default: ''
+        },
+        fallbackImage: {
+            type: String,
+            default: ''
         }
     },
     data () {
         return {
             isToggleSelected: false
         };
     },
+    computed: {
+        /**
+         * Returns a boolean which adds or removes
+         * the 'presentation' role from the image
+         *
+         */
+        addPresentationRole () {
```

Comment:
```suggestion
        isPresentationRole () {
```
The name with the action in it is a bit misleading for me. I would name booleans with "is" prefix. It's a bit more clear that this function doesn't add anything, but it just returns a boolean value.